c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -286,7 +286,6 @@ TEMPLATES_FILES = $(shell find $(PACKAGE)/templates -type f -print)
 
 # Templates
 TEMPLATE_EXCLUDE += .build print/templates \
-	templatecache.js.mako \
```

Comment:
I don't understand. I thought `TEMPLATE_EXCLUDE` was a list a files that shouldn't be passed to `c2c.template`. And we don't want to pass `templatecache.js.mako` to `c2c.template`.

Am I not understanding this correctly?